game/resource/addon_english.txt

```diff
@@ -280,164 +280,164 @@
         "Script_4_Communication_17" "Head to the next marker. I won't tell you where it is, so use your minimap and camera to find it. We will meet you there!"
         "Script_4_RTZ_getaway" "Later *Bleep*"
         "Script_4_RTZ_foundme" "Shit, they found me."
         "Script_4_RTZ_cya" "See ya!"
         "Script_4_RTZ_death" "Yea I dont know I'm done."
         "Script_4_RTZ_pain" "Ugh"
         "Script_4_mason_mad" "(Breaks items)"
-        "Script_5_Opening_1" "welcome, to the bounty rune spawn! Yes there are more things that spawn!"
-        "Script_5_Opening_2" "bounty runes spawn every 5 minutes, including the beginning of the game at the 0 mark. Here are where they spawn"
+        "Script_5_Opening_1" "Welcome, to the bounty rune spawn! Yes there are more things that spawn!"
+        "Script_5_Opening_2" "Bounty runes spawn every 5 minutes, including the beginning of the game at the 0 mark. Here are where they spawn."
         "Script_5_Opening_3" "Here we are, then here (the radiant jungle). Your opponents also have 2 spawns, here (the dire jungle) and here (the dire ancients)."
         "Script_5_Opening_4" "Bounty runes, when picked up, give your entire team a small amount of gold. But it adds up! As the game goes on, the runes are worth more and more. Making sure you are collecting your side’s bounty runes is a way to ensure your team stays in the game."
-        "Script_5_Opening_5" "If you manage to steal the enemy’s runes, and get all 4, your entire team gets a much bigger advantage over the enemy, as you are all making more money while the enemy gets less."
-        "Script_5_Opening_6" "But believe it or not, this is not the only type of rune, dear student. Lets head over here!"
-        "Script_5_Opening_7" "This is called a POWER RUNE spawn. (PLAYS GUITAR SOLO) These are powerups that spawn every 2 minutes, starting at minute 4. These dramatically change your character to be more powerful for a short time. There are many types of these runes, and I don't feel like telling you about all of them. But we’ll show you with the help of the power rune rangers...which are not actually a real thing."
+        "Script_5_Opening_5" "If you manage to steal the enemy’s runes and get all 4, your entire team gets a much bigger advantage over the enemy, as you are all making more money while the enemy gets less."
+        "Script_5_Opening_6" "But believe it or not, this is not the only type of rune, dear student. Let's head over here!"
+        "Script_5_Opening_7" "This is called a POWER RUNE spawn. These are powerups that spawn every 2 minutes, starting at minute 4. These dramatically change your character to be more powerful for a short time. There are many types of these runes, and I don't feel like telling you about all of them. But we’ll show you with the help of the power rune rangers... which are not actually a real thing."
```

Comment:
Change "powerups" to "power-ups". 
Change "2" to "two".
Change "power rune rangers" to "Power Rune Rangers".